Add a note about temp files stored on non-encrypted disk

SaloniSonpal opened the pull request:
Adding an exception for temporary files which are stored on non-encrypted disk.

azurecla commented:
Hi __@punitganshani__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

SaloniSonpal committed:
Add a note about temp files being not encrypted

Adding an exception for temporary files which are stored on non-encrypted disk.
